[2022-04-28 06:04:21,122] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [queued]>
[2022-04-28 06:04:21,173] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [queued]>
[2022-04-28 06:04:21,175] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-28 06:04:21,176] {taskinstance.py:1250} INFO - Starting attempt 1 of 2
[2022-04-28 06:04:21,177] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-28 06:04:21,231] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 1999-01-01 00:00:00+00:00
[2022-04-28 06:04:21,240] {standard_task_runner.py:52} INFO - Started process 1582 to run task
[2022-04-28 06:04:21,251] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcp_dag', 'local_to_gcs_task', 'scheduled__1999-01-01T00:00:00+00:00', '--job-id', '519', '--raw', '--subdir', 'DAGS_FOLDER/ingest_to_gcs_dag.py', '--cfg-path', '/tmp/tmpvoj3znx3', '--error-file', '/tmp/tmpus8ua6u7']
[2022-04-28 06:04:21,256] {standard_task_runner.py:80} INFO - Job 519: Subtask local_to_gcs_task
[2022-04-28 06:04:21,801] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [running]> on host d4721d223bdd
[2022-04-28 06:04:22,167] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-28 06:04:22,438] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcp_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=1999-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__1999-01-01T00:00:00+00:00
[2022-04-28 06:04:24,001] {python.py:175} INFO - Done. Returned value was: None
[2022-04-28 06:04:24,113] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcp_dag, task_id=local_to_gcs_task, execution_date=19990101T000000, start_date=20220428T060421, end_date=20220428T060424
[2022-04-28 06:04:24,196] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-28 06:04:24,842] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-29 02:34:43,146] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [queued]>
[2022-04-29 02:34:43,192] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [queued]>
[2022-04-29 02:34:43,194] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-29 02:34:43,196] {taskinstance.py:1250} INFO - Starting attempt 1 of 2
[2022-04-29 02:34:43,197] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-29 02:34:43,243] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 1999-01-01 00:00:00+00:00
[2022-04-29 02:34:43,253] {standard_task_runner.py:52} INFO - Started process 5177 to run task
[2022-04-29 02:34:43,262] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcp_dag', 'local_to_gcs_task', 'scheduled__1999-01-01T00:00:00+00:00', '--job-id', '698', '--raw', '--subdir', 'DAGS_FOLDER/ingest_to_gcs_dag.py', '--cfg-path', '/tmp/tmpusgbw89o', '--error-file', '/tmp/tmpc1q9nk74']
[2022-04-29 02:34:43,265] {standard_task_runner.py:80} INFO - Job 698: Subtask local_to_gcs_task
[2022-04-29 02:34:43,644] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [running]> on host 6fd0ba274d58
[2022-04-29 02:34:43,910] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-29 02:34:44,098] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcp_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=1999-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__1999-01-01T00:00:00+00:00
[2022-04-29 02:34:45,567] {python.py:175} INFO - Done. Returned value was: None
[2022-04-29 02:34:45,671] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcp_dag, task_id=local_to_gcs_task, execution_date=19990101T000000, start_date=20220429T023443, end_date=20220429T023445
[2022-04-29 02:34:45,752] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-29 02:34:45,985] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-05-02 07:54:01,773] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [queued]>
[2022-05-02 07:54:01,818] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [queued]>
[2022-05-02 07:54:01,819] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-05-02 07:54:01,821] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-05-02 07:54:01,822] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-05-02 07:54:01,868] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 1999-01-01 00:00:00+00:00
[2022-05-02 07:54:01,878] {standard_task_runner.py:52} INFO - Started process 9121 to run task
[2022-05-02 07:54:01,885] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcp_dag', 'local_to_gcs_task', 'scheduled__1999-01-01T00:00:00+00:00', '--job-id', '851', '--raw', '--subdir', 'DAGS_FOLDER/ingest_to_gcs_dag.py', '--cfg-path', '/tmp/tmp6u_d2oi1', '--error-file', '/tmp/tmp5a_9xcsn']
[2022-05-02 07:54:01,889] {standard_task_runner.py:77} INFO - Job 851: Subtask local_to_gcs_task
[2022-05-02 07:54:02,413] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [running]> on host ae42322ae538
[2022-05-02 07:54:02,687] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-05-02 07:54:02,891] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcp_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=1999-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__1999-01-01T00:00:00+00:00
[2022-05-02 07:54:03,631] {python.py:175} INFO - Done. Returned value was: None
[2022-05-02 07:54:03,736] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcp_dag, task_id=local_to_gcs_task, execution_date=19990101T000000, start_date=20220502T075401, end_date=20220502T075403
[2022-05-02 07:54:03,805] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-02 07:54:04,208] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-05-02 09:15:54,019] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [queued]>
[2022-05-02 09:15:54,066] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [queued]>
[2022-05-02 09:15:54,068] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-05-02 09:15:54,069] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-05-02 09:15:54,070] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-05-02 09:15:54,114] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 1999-01-01 00:00:00+00:00
[2022-05-02 09:15:54,124] {standard_task_runner.py:52} INFO - Started process 2372 to run task
[2022-05-02 09:15:54,132] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcp_dag', 'local_to_gcs_task', 'scheduled__1999-01-01T00:00:00+00:00', '--job-id', '969', '--raw', '--subdir', 'DAGS_FOLDER/ingest_to_gcs_dag.py', '--cfg-path', '/tmp/tmpiksqeief', '--error-file', '/tmp/tmpmguhj793']
[2022-05-02 09:15:54,135] {standard_task_runner.py:77} INFO - Job 969: Subtask local_to_gcs_task
[2022-05-02 09:15:54,508] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [running]> on host 98c324cf1bb6
[2022-05-02 09:15:54,771] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-05-02 09:15:54,965] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcp_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=1999-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__1999-01-01T00:00:00+00:00
[2022-05-02 09:15:55,737] {python.py:175} INFO - Done. Returned value was: None
[2022-05-02 09:15:55,872] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcp_dag, task_id=local_to_gcs_task, execution_date=19990101T000000, start_date=20220502T091554, end_date=20220502T091555
[2022-05-02 09:15:55,939] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-02 09:15:56,162] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-05-02 15:57:13,713] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [queued]>
[2022-05-02 15:57:13,758] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [queued]>
[2022-05-02 15:57:13,760] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-05-02 15:57:13,761] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-05-02 15:57:13,763] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-05-02 15:57:13,810] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 1999-01-01 00:00:00+00:00
[2022-05-02 15:57:13,822] {standard_task_runner.py:52} INFO - Started process 1538 to run task
[2022-05-02 15:57:13,831] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcp_dag', 'local_to_gcs_task', 'scheduled__1999-01-01T00:00:00+00:00', '--job-id', '1071', '--raw', '--subdir', 'DAGS_FOLDER/ingest_to_gcs_dag.py', '--cfg-path', '/tmp/tmp58jvdta4', '--error-file', '/tmp/tmp4obr8oyr']
[2022-05-02 15:57:13,835] {standard_task_runner.py:77} INFO - Job 1071: Subtask local_to_gcs_task
[2022-05-02 15:57:14,213] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [running]> on host 9f5021834e17
[2022-05-02 15:57:14,480] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-05-02 15:57:14,680] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcp_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=1999-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__1999-01-01T00:00:00+00:00
[2022-05-02 15:57:15,513] {python.py:175} INFO - Done. Returned value was: None
[2022-05-02 15:57:15,620] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcp_dag, task_id=local_to_gcs_task, execution_date=19990101T000000, start_date=20220502T155713, end_date=20220502T155715
[2022-05-02 15:57:15,701] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-02 15:57:15,921] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-05-02 16:32:13,103] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [queued]>
[2022-05-02 16:32:13,153] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [queued]>
[2022-05-02 16:32:13,155] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-05-02 16:32:13,158] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-05-02 16:32:13,161] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-05-02 16:32:13,210] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 1999-01-01 00:00:00+00:00
[2022-05-02 16:32:13,221] {standard_task_runner.py:52} INFO - Started process 6095 to run task
[2022-05-02 16:32:13,231] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcp_dag', 'local_to_gcs_task', 'scheduled__1999-01-01T00:00:00+00:00', '--job-id', '1177', '--raw', '--subdir', 'DAGS_FOLDER/ingest_to_gcs_dag.py', '--cfg-path', '/tmp/tmp8jqdyp3q', '--error-file', '/tmp/tmpyyit8yhb']
[2022-05-02 16:32:13,235] {standard_task_runner.py:77} INFO - Job 1177: Subtask local_to_gcs_task
[2022-05-02 16:32:13,693] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [running]> on host 9f5021834e17
[2022-05-02 16:32:13,962] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-05-02 16:32:14,166] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcp_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=1999-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__1999-01-01T00:00:00+00:00
[2022-05-02 16:32:14,978] {python.py:175} INFO - Done. Returned value was: None
[2022-05-02 16:32:15,082] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcp_dag, task_id=local_to_gcs_task, execution_date=19990101T000000, start_date=20220502T163213, end_date=20220502T163215
[2022-05-02 16:32:15,152] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-02 16:32:15,393] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-05-02 16:47:40,893] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [queued]>
[2022-05-02 16:47:40,951] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [queued]>
[2022-05-02 16:47:40,953] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-05-02 16:47:40,954] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-05-02 16:47:40,956] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-05-02 16:47:41,007] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 1999-01-01 00:00:00+00:00
[2022-05-02 16:47:41,018] {standard_task_runner.py:52} INFO - Started process 8911 to run task
[2022-05-02 16:47:41,029] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcp_dag', 'local_to_gcs_task', 'scheduled__1999-01-01T00:00:00+00:00', '--job-id', '1282', '--raw', '--subdir', 'DAGS_FOLDER/ingest_to_gcs_dag.py', '--cfg-path', '/tmp/tmp7lo99kfe', '--error-file', '/tmp/tmpkh1t9uoh']
[2022-05-02 16:47:41,032] {standard_task_runner.py:77} INFO - Job 1282: Subtask local_to_gcs_task
[2022-05-02 16:47:41,498] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcp_dag.local_to_gcs_task scheduled__1999-01-01T00:00:00+00:00 [running]> on host 9f5021834e17
[2022-05-02 16:47:41,771] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-05-02 16:47:41,987] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcp_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=1999-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__1999-01-01T00:00:00+00:00
[2022-05-02 16:47:42,844] {python.py:175} INFO - Done. Returned value was: None
[2022-05-02 16:47:42,965] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcp_dag, task_id=local_to_gcs_task, execution_date=19990101T000000, start_date=20220502T164740, end_date=20220502T164742
[2022-05-02 16:47:43,027] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-02 16:47:43,282] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
